cnf-certification-test/performance/suite.go

```diff
@@ -232,30 +233,86 @@ func testSchedulingPolicyInCPUPool(env *provider.TestEnvironment,
 	testhelper.AddTestResultReason(compliantContainersPids, nonCompliantContainersPids, tnf.ClaimFilePrintf, ginkgo.Fail)
 }
 
+func getExecProbesCmds(c *provider.Container) map[string]bool {
+	cmds := map[string]bool{}
+
+	if c.LivenessProbe != nil && c.LivenessProbe.Exec != nil {
+		for _, cmd := range c.LivenessProbe.Exec.Command {
+			cmds[cmd] = true
```

Comment:
Here we are storing each command component in the map, e.g. "cat test.yaml" will result in 2 entries in the map: "cat" and "test.yaml". Later, testing will never match these command bits since they are compared with the full command line from ps. 
Instead, reconstruct the full command line here minus any spaces, carriage return or semi colons and compare it with the output from ps (also stripped from semi colons, spaces, carriage return, etc)